google/cloud/documentai_toolbox/wrappers/entity.py

```diff
@@ -26,44 +27,73 @@
 
 @dataclasses.dataclass
 class Entity:
-    r"""Represents a wrapped documentai.Document.Entity.
+    """Represents a wrapped `documentai.Document.Entity`.
 
     Attributes:
+        shard_index (int):
```

Comment:
Since these wrapper classes are "public", let's acknowledge that this change of adding a new required argument (and in other places) is a breaking change in the commit message.

Reply:
No longer applicable